portal-api/src/v1/controllers/bonds.controller.js

```diff
@@ -89,6 +90,7 @@ const facilityStageFields = (bond) => {
     modifiedBond['coverEndDate-month'] = null;
     modifiedBond['coverEndDate-year'] = null;
     modifiedBond.uniqueIdentificationNumber = null;
+    modifiedBond.hasBeenIssued = false;
```

Comment:
Great work,a looking great! Can we update or add API test for this (and loan), it's in:

```
portal-api/api-tests/v1/bonds/bonds.api-test.js
portal-api/api-tests/v1/loans/loans.api-test.js
```

Or I can do this if you'd like?

There's a couple of describe blocks, first one is this:

```
describe('when a bond has req.body.facilityStage as `Issued`', () => { ...
```

Probably need to update this and the one below, for bonds and loans 
PS sorry for the API tests on this, they're a bit long/complicated.


Side note: we could simplify some TFM graphQL mappings for facility with this in place, no need to be done right now

Reply:
thanks @ttbarnes 
I think you have more experience with these tests. if it's not too much work, can I pass it over to you?

Reply:
@ttbarnes I've added the tests now - let me know if this is what you had in mind

Reply:
I was just pulled the branch to do this and yes, perfect :)